pkg/app/gen/html.go

```diff
@@ -2188,6 +2194,15 @@ func writeAttrFunction(w io.Writer, a attr, t tag, isInterface bool) {
 			}`, "%v")
 		}
 
+	case "key|value":
+		fmt.Fprintf(w, `%s(k string, v interface{}) HTML%s`, a.Name, t.Name)
```

Comment:
We usually refer to an attribute by its name => https://developer.mozilla.org/en-US/docs/Web/API/Element/setAttribute

The param should be `(n string, v interface{})`